modules/gateway/consts.go

```diff
@@ -68,6 +77,14 @@ var (
 		Dev:      15,
 		Testing:  10,
 	}).(int)
+
+	// quickPruneListLen defines the number of nodes that the gateway must have
+	// to be pruning nodes quickly from the node list.
+	quickPruneListLen = build.Select(build.Var{
+		Standard: 250,
+		Dev:      40,
+		Testing:  20,
+	}).(int)
```

Comment:
best practice to explicitly cast these to `int` (i.e. `int(250)`, `int(40)`). Otherwise someone might come along and think they could change the `int` at the bottom to a `uint64` or something, which they can't -- it'll panic.